plugins/modules/pnp_intent.py

```diff
@@ -566,29 +581,31 @@ def get_have(self):
                     params=self.want.get("image_params"),
                 )
                 image_list = image_response.get("response")
-                self.log(str(image_response))
+                self.log("Image details obtained from the API 'get software image details' are {0}".format(str(image_response)), "DEBUG")
```

Comment:
We can specify the API name directly as the APIs are opensource. 
self.log("Image details obtained from the API 'get_software_image_details': {0}".format(str(image_response)), "DEBUG")

Reply:
Resolved